Native Mode & Bandwidth: Update overview-native-mode.md (#594)
MicrosoftDocs/Viva: Viva/engage/overview-native-mode.md

 > [!NOTE]
> Native Mode is strongly recommended for reasons of security, compliance, and Microsoft 365 integration.

 > [!NOTE]
> There are no bandwidth requirements for Native Mode migration because you are essentially moving from a separate Yammer site to Microsoft 365 and SharePoint Online, nothing is downloaded. The only consideration is the SharePoint Online quota if Yammer has been extensively used.

> 

Reviewer: ```suggestion
```

> > There are no bandwidth requirements for the Native mode migration. Essentially you are moving between Yammer separate site to Microsoft 365/SharePoint Online, nothing is downloaded. The only consideration to have in for SharePoint Online quota, in case Yammer was heavily being used.

Reviewer: ```suggestion
> There are no bandwidth requirements for Native Mode migration because you are essentially moving from a separate Yammer site to Microsoft 365 and SharePoint Online, nothing is downloaded. The only consideration is the SharePoint Online quota should Yammer have been extensively used.
```
